tensorboard/uploader/uploader.py

```diff
@@ -408,6 +442,26 @@ def _create_point(self, tag_proto, event, value):
         return point
 
 
+class RequestLogger:
```

Comment:
Can this be simplified with `contextlib`? I think that

```python
@contextlib.contextmanager
def _request_logger(request):
    upload_start_time = time.time()
    request_bytes = request.ByteSize()
    logger.info("Trying request of %d bytes", request_bytes)
    yield
    upload_duration_secs = time.time() - upload_start_time
    logger.info(...)
```

should do it, but I would feel more comfortable with that assertion if
`RequestLogger` had some tests.

Reply:
Oh neat, I didn't know about `contextlib`.  This works fine.  Verified manually; seems to me too basic to bother with a test.